.gitbook/developers-evm/evm-integrations-cheat-sheet.mdx

```diff
@@ -0,0 +1,120 @@
+
+Note: Do **not** use inEVM, as that has been **deprecated**.
+
+# Explorers
+
+- [https://blockscout.injective.network/](https://blockscout.injective.network/) → Can only see EVM transactions
+- [https://injscan.com/](https://injscan.com/) → Can see both EVM and Cosmos transactions
+
+Reference: [EVM Network Information | Injective | Docs](/developers-evm/network-information#injective-evm-mainnet) 
+
+# RPC Endpoints
+
+- Recommended for integrations partners
+    - Customizable for needs of applications
+    - Customizable rate limits, and no cut off for historical data queries
+    - Freemium/ paid options
+    - How to connect:
+        - QuickNode: [Injective RPC Node Endpoints, APIs & Tools | QuickNode](https://www.quicknode.com/chains/inj) 
+        - Thirdweb: [Injective EVM](https://thirdweb.com/injective)
+- Not recommended for integrations partners
+    - Not customizable for needs of applications
+    - Heavy rate limiting, and cut off for historical data queries
+    - Free option
+    - How to connect:
+        - JSON-RPC Endpoint: `https://sentry.evm-rpc.injective.network/`
+        - WS Endpoint: `wss://sentry.evm-ws.injective.network`
+    
+Reference: [EVM Network Information | Injective | Docs](/developers-evm/network-information#injective-evm-mainnet) 
+
+Note: Do **not** use inEVM, as that has been **deprecated**.
+
+# Contract Addresses
+
+|   |   |   |
+|---|---|---|
+|USDT|MTS USDT|`0x88f7F2b685F9692caf8c478f5BADF09eE9B1Cc13`|
+|wETH|wrapped ETH|`0x83A15000b753AC0EeE06D2Cb41a69e76D0D5c7F7`|
+|wINJ|wrapped INJ|`0x0000000088827d2d103ee2d9A6b781773AE03FfB`|
+|USDC||`0x2a25fbD67b3aE485e461fe55d9DbeF302B7D3989`|
+|MultiCall||`0xcA11bde05977b3631167028862bE2a173976CA11`|
+
+Please use the reference page below as the canonical source.
+
+Reference: [EVM Network Information | Injective | Docs](/developers-evm/network-information#injective-evm-mainnet) 
+
+# Bridges
+
+[Injective Bridge](https://bridge.injective.network)
+
+Guides:
+- [How To Bridge From Ethereum To Injective Using Metamask](https://injective.com/blog/how-to-bridge-from-ethereum-to-injective-using-metamask/)
+- [How To Bridge From Solana To Injective Using Phantom](https://injective.com/blog/how-to-bridge-from-solana-to-injective-using-phantom/)
+- [How to Bridge To Injective Using Wormhole](https://injective.com/blog/how-to-bridge-to-injective-using-wormhole/)
+- [How To Bridge From Cosmos To Injective Using Keplr](https://injective.com/blog/how-to-bridge-from-cosmos-to-injective-using-keplr/)
+
+# Data
+
+Explorers
+
+- Explorer: [`blockscout.injective.network`](https://blockscout-api.injective.network/api)
+- Explorer API: [`blockscout-api.injective.network/api`](https://blockscout-api.injective.network/api)
+
+Oracles
+
+- API3: 
+    - Price feeds: [Api3 Market | Injective EVM](https://market.api3.org/injective)
+- Pyth:
+    - Announcement: [Pyth Mainnet Launches on Injective](https://injective.com/blog/pyth-mainnet-launches-on-injective/) 
```

Comment:
This references the CosmWasm integration. We should link people directly to the Pyth docs for CosmWasm & EVM Injective.